WebContent/index.html

```diff
@@ -16,6 +16,33 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=0, maximum-scale=1, minimum-scale=1">
     <title>JES Explorer</title>
 
+    <script>
+      if(exports){
+        var ZoweZLUX_tempExports = exports;
+      }
+      var exports = {"__esModule": true};
+    </script>
+    <script type="text/javascript" src="../../../../../lib/org.zowe.zlux.logger/0.9.0/logger.js"></script>
```

Comment:
I think we talked about dynamically loading this? We want to avoid loading it if we're not in the desktop.

I'm also very concerned about the path with versions in it, how will we know when the path needs updating? Maybe we should discuss with web squad on strategy for reliably pulling in the logger file?

Reply:
right now we pulled in script from our project, we may write javascript file to load other javascript file and put conditional logic there, looks clean now, we can definitely explore this possibility in separate PR

Reply:
> I'm also very concerned about the path with versions in it, how will we know when the path needs updating? Maybe we should discuss with web squad on strategy for reliably pulling in the logger file?

For path, its local now, as we pulled script into project, 

1) we can explore other options like putting validate script when installing it as iframe plugin
2) Hard coding relative path - 
between this 
/1.8.0-runtime/components/explorer-jes/bin/app/index.html 
and these 
./1.8.0-runtime/components/app-server/share/zlux-app-manager/bootstrap/web/iframe-adapter.js
./1.8.0-runtime/components/app-server/share/zlux-shared/src/logging/logger.js